ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json

```diff
@@ -73,7 +73,7 @@
     "PageID": "SingleCareOrder",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND singleCareOrder_label!=\"\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND orderTypeAndDocument.type!=\"\" AND singleCareOrder_label!=\"\"",
```

Comment:
is it needed? orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" should be enough